selftests/cyclical_deps

```diff
@@ -18,6 +18,14 @@ import sys
 import subprocess
 import tempfile
 
+# it doesn't make sense to go any further on Python 3 because snakefood
+# won't be available anyway.  it's not fair to say this check failed, so
+# let's exit successfully.  the check should be performend when running
+# on Python 2.
+if sys.version_info[0] == 3:
+    sys.exit(0)
+
```

Comment:
So we need a card to check whether we want to keep this check, rewriting it to Python 3, or just drop it.

Reply:
Agreed: https://trello.com/c/rfSAXdyi/1172-find-python-3-replacement-for-snakefood-networkx-checks